intake/catalog/tests/test_parameters.py

```diff
@@ -0,0 +1,138 @@
+import os
+import pytest
+from intake.catalog.local import LocalCatalogEntry, UserParameter
+from intake.source.base import DataSource
+
+
+class NoSource(DataSource):
+
+    def __init__(self, **kwargs):
+        self.kwargs = kwargs
+
+
+driver = 'intake.catalog.tests.test_parameters.NoSource'
+
+
+def test_simplest():
+    e = LocalCatalogEntry('', '', driver, args={'arg1': 1})
+    s = e()
+    assert s.kwargs['arg1'] == 1
+
+
+def test_parameter_default():
+    up = UserParameter('name', default='oi')
+    e = LocalCatalogEntry('', '', driver, args={'arg1': "{{name}}"},
+                          parameters=[up])
+    s = e()
+    assert s.kwargs['arg1'] == 'oi'
+
+
+def test_maybe_par_env():
```

Comment:
I don't understand this name

Reply:
maybe fill in parameter default from the env, depending on getenv

Reply:
I can add exactly that text as a comment

Reply:
Sorry I meant to generalize this comment. I think you could stand to have longer test names that are more explicit.